src/main/java/com/puppycrawl/tools/checkstyle/checks/coding/IllegalThrowsCheck.java

```diff
@@ -146,11 +146,9 @@ public void visitToken(DetailAST detailAST) {
         if (!isIgnorableMethod(methodDef)) {
             DetailAST token = detailAST.getFirstChild();
             while (token != null) {
-                if (token.getType() != TokenTypes.COMMA) {
```

Comment:
`FullIdent.createFullIdent` just creates a text string with `,`. There is no way to add a comma to `illegalClassNames` naturally as all are list strings are comma separated.